addons/misc/functions/fnc_constructstripExplosivesDialog.sqf

```diff
@@ -89,7 +89,7 @@ else {
 	_controls pushBack ["CHECKBOX", [localize "STR_CROWSZA_Misc_strip_explosives_whole_group", localize "STR_CROWSZA_Misc_strip_explosives_whole_group_tooltip"], _controlValues select 0, _default];
 };
 _controls pushBack ["CHECKBOX", localize "STR_CROWSZA_Misc_strip_explosives_ignore_players", _controlValues select 1, _default];
-_controls pushBack ["TOOLBOX", [localize "STR_CROWSZA_Misc_strip_explosives_item_type", parseText localize "STR_CROWSZA_Misc_strip_explosives_item_type_tooltip"], [_controlValues select 2, 1, 5, [localize "STR_CROWSZA_Misc_strip_explosives_signals", localize "STR_CROWSZA_Misc_strip_explosives_grenades", localize "STR_CROWSZA_Misc_strip_explosives_explosives", localize "STR_CROWSZA_Misc_strip_explosives_ugl", localize "STR_CROWSZA_Misc_strip_explosives_launchers"]], _default];
+_controls pushBack ["TOOLBOX", [localize "STR_CROWSZA_Misc_strip_explosives_item_type", (localize "STR_CROWSZA_Misc_strip_explosives_item_type_tooltip") regexReplace ["<br/>",endl]], [_controlValues select 2, 1, 5, [localize "STR_CROWSZA_Misc_strip_explosives_signals", localize "STR_CROWSZA_Misc_strip_explosives_grenades", localize "STR_CROWSZA_Misc_strip_explosives_explosives", localize "STR_CROWSZA_Misc_strip_explosives_ugl", localize "STR_CROWSZA_Misc_strip_explosives_launchers"]], _default];
```

Comment:
Same questions as before with regex. Just remind me if you found out the parseText didn't work, or if something else caused this change? 
I seem to recall it worked, but I also had to test all of the localisations at once, so could have missed it not working.